### ISSUE TITLE
YYC only crash, new in 2024.8

### ISSUE THREAD

**Issue by KormexGit** [YoYoGames/GameMaker-Bugs#6759]
### Description
Unfortunately I am not able to narrow this down to something specific since it's happening with the Input library, and I'm not familiar with it's interals. This crash is definitely new, since it works just fine in 2024.6, and also works fine in VM on 2024.8. It crashes instantly on launch in YYC only.

### Steps To Reproduce
Launch any project with the Input library in YYC, and it will crash instantly. Do the same in 2024.6 or VM, and it works fine.

### Which version of GameMaker are you reporting this issue for?
IDE v2024.800.0.593 Runtime v2024.800.0.617

### Which operating system(s) are you seeing the problem on?
Windows 10.0.22631.0

[ccc21144-73ce-48b0-85f4-51f91f2bf45b](https://api.gamemaker.io/api/github/downloads/ccc21144-73ce-48b0-85f4-51f91f2bf45b)

**Comment by KormexGit** [comment]
After talking to alynne, it was found that this message prints true in VM, but false in YYC, so something seems to be broken with constructors
```
    _global.__virtual_background = input_virtual_create();
	show_message(is_instanceof(_global.__virtual_background, __input_class_virtual))
	_global.__virtual_background.__set_as_background();
	```

**Comment by rwkay** [comment]
I have isolated the issue and fixed it internally - my bad, I fixed a different issue and part of that fix changed some of the logic which led to this one... thanks for reporting so quickly

**Comment by rwkay** [comment]
Fixed in 2024.8

**Comment by mgeddesGM** [comment]
verified as of  IDE v2024.800.0.595  Runtime v2024.800.0.619
